x-pack/plugins/xpack_main/public/hacks/check_xpack_info_change.js

```diff
@@ -81,18 +81,21 @@ module.factory('checkXPackInfoChange', ($q, Private) => {
       return handleResponse(response);
     }
 
-    const currentSignature = response.headers('kbn-xpack-sig');
-    const cachedSignature = xpackInfoSignature.get();
+    // If another interceptor has rejected a request, then headers will be undefined.
+    if (response.headers) {
```

Comment:
Without this condition, a blocked request to `api/ui_metric` will throw an error on ln 86.